mk/package-and-sign.sh

```diff
@@ -52,6 +52,8 @@ mkdir_clean()
   rm -rf $1 && mkdir -p $1
 }
 
+mkdir_clean ${OUTPUT_DIR}
```

Comment:
@MihaelaStoica this line should not be necessary because the output folder should have already been created by the Jenkinsfile in the 'Create manifest' stage. Moreover this line clears the created manifest. Did you add it because you noticed a different behaviour?